internal/core/operations/update_scheduler/update_scheduler_executor.go

```diff
@@ -32,16 +32,19 @@ import (
 	"github.com/topfreegames/maestro/internal/core/entities/operation"
 	"github.com/topfreegames/maestro/internal/core/operations"
 	"github.com/topfreegames/maestro/internal/core/services/room_manager"
-	"github.com/topfreegames/maestro/internal/core/services/scheduler_manager"
 	"go.uber.org/zap"
 )
 
+type SchedulerManager interface {
```

Comment:
@lyracampos why did you create this interface?  regardless of the reason, I don't see this file being the proper place to define I, right?

Reply:
@GuilhermBrSp,It was necessary to create this interface to avoid circular dependency/reference between update scheduler and scheduler manager.

yes, i agree that this interface can be wrote in a specific file. I'm not sure about where because we don't have this kind of implementation yet. 
I note that interfaces are placed in 'internal/core/ports' but in my point of view this new interface aren't a port interface concept.

I saw in some github projects open source that devs put interface into same folder that implementation ex: update_scheduler.go and update_scheduler_impl.go but for me it smeels like java implementations hahaha.

my suggestion here is create new folder in /internal/core/services/interfaces to put interfaces that abstract services implementations. What do you think?

Reply:
@lyracampos  I got the circular dependency problem, i strongly agree on creating domain service interfaces for two reasons:
- Solve this type of circular dependency problem
- Those interfaces will enable us to create mocks for these services and use them in handlers tests. Currently, in our handler tests, we have to mock all dependencies of the services that it uses, generating a level of mocking that shouldn't exist in those scenarios.

Regarding where we should declare those interfaces, I like your suggestion to place it inside `/internal/core/services/interfaces`